client/src/main/java/org/apache/rocketmq/client/impl/consumer/ConsumeMessageConcurrentlyService.java

```diff
@@ -284,6 +284,13 @@ public void processConsumeResult(
                 List<MessageExt> msgBackFailed = new ArrayList<>(consumeRequest.getMsgs().size());
                 for (int i = ackIndex + 1; i < consumeRequest.getMsgs().size(); i++) {
                     MessageExt msg = consumeRequest.getMsgs().get(i);
+                    // Maybe message is expired and cleaned, just ignore it.
+                    if (!consumeRequest.getProcessQueue().hasMessage(msg)) {
+                        log.info("Message does not exist in process queue, skip to send it back, topic={}, "
```

Comment:
The logging content looks ambiguous...

`Message does not exist in process queue, skip to send it back, topic={},` -->
`Message is not found in its process queue; skip send-back-procedure.`